pkg/math/integer.go

```diff
@@ -0,0 +1,23 @@
+package math
```

Comment:
Can we add a package comment explaining why this package is needed?

Reply:
yeah. https://github.com/zeta-chain/node/pull/3306/commits/24d12ae23bb5cd37092394a76b2ef55a8aadce94